assets/css/styleguide.scss

```diff
@@ -128,7 +130,16 @@ $site-top:           124px;
   display: inline;
   float: left;
   margin-top: -4px;
-  padding-left: 20px;
+  padding-top: 15px;
```

Comment:
Lets use rems for anything more than a few pixels per the 18F coding styleguide.